test/integration/serverless/snapshots/log-csharp

```diff
@@ -1,170 +1,182 @@
 [
   {
     "message": {
-      "message": "START RequestId: XXX Version: $LATEST",
+      "message": "END RequestId: XXX",
       "lambda": {
-        "arn": "arn:aws:lambda:eu-west-1:425362996713:function:integration-tests-extension-STAGE-log-csharp",
+        "arn": "arn:aws:lambda:eu-west-1:425362996713:function:integration-tests-extension-XXXXXX-log-csharp",
```

Comment:
I changed `STAGE` to `XXXXXX` throughout. The other two telemetry types were changing the stage to `XXXXXX`, so I feel like this keeps things a bit more consistent.